apps/andi/test/unit/andi/dataset_store_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Andi.DatasetStoreTest do
+  use ExUnit.Case
+  use Placebo
+
+  import SmartCity.Event, only: [dataset_update: 0]
+
+  alias SmartCity.TestDataGenerator, as: TDG
+  alias Andi.DatasetStore
+
+  @brook_instance :andi
+
+  describe "get_all/0" do
+    test "retrieves all events from Brook" do
+      dataset1 = TDG.create_dataset(%{})
+      dataset2 = TDG.create_dataset(%{})
+      expected_datasets = {:ok, [dataset1, dataset2]}
+      Brook.Event.send(@brook_instance, dataset_update(), :dataset, dataset1)
+      Brook.Event.send(@brook_instance, dataset_update(), :dataset, dataset2)
```

Comment:
I don't think you need these calls to Brook.Event.send anymore (since you're mocking out get_all_values).